acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/DefaultAwsSecretsManagerParameters.java

```diff
@@ -15,24 +15,27 @@
 import tech.pegasys.web3signer.signing.config.AwsAuthenticationMode;
 import tech.pegasys.web3signer.signing.config.AwsSecretsManagerParameters;
 
+import java.util.Collection;
+
 public class DefaultAwsSecretsManagerParameters implements AwsSecretsManagerParameters {
 
   private final AwsAuthenticationMode authenticationMode;
   private final String region;
   private final String accessKeyId;
   private final String secretAccessKey;
-  private final String secretName;
+
+  private final Collection<String> prefixFilter;
 
   public DefaultAwsSecretsManagerParameters(
       final String region,
       final String accessKeyId,
       final String secretAccessKey,
-      final String secretName) {
+      final Collection<String> prefixFilter) {
```

Comment:
What about the other filters? Are they also tested?

Reply:
This class has been removed in favor of `tech.pegasys.web3signer.signing.config.AwsSecretsManagerParametersBuilder` which exists in signing testFixture and can be utilized in AT dsl.